Makefile

```diff
@@ -267,10 +267,14 @@ bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metada
 ##@ Bundle Creation Addition
 ## Some addition to bundle creation in the bundle
 .PHONY: bundle-update
+DEFAULT_ICON_BASE64 := $(shell base64 --wrap=0 ./config/assets/medik8s_blue_icon.png)
+export ICON_BASE64 ?= ${DEFAULT_ICON_BASE64}
 export CSV="./bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml"
-bundle-update: ## Update containerImage and createdAt
+
+bundle-update: ## Update containerImage, createdAt, icon, and displayName fields in the bundle's CSV, then validate the bundle directory
 	sed -r -i "s|containerImage: .*|containerImage: $(IMG)|;" ${CSV}
 	sed -r -i "s|createdAt: .*|createdAt: \"`date '+%Y-%m-%d %T'`\"|;" ${CSV}
+	sed -r -i "s|base64data:.*|base64data: ${ICON_BASE64}|;" ${BUNDLE_CSV}
```

Comment:
this seems a typo `s/BUNDLE_CSV/CSV`
Otherwise it is ok to use `BUNDLE_CSV` everywhere, if you think it is more descriptive

Reply:
That's indeed a mistake.
Basically there are two CSV in the code, one under `config` and one under `bundle`, and that's why I thought of mentioning it. But we are under a section of  `Bundle Creation Addition` so it should be clear that the CSV is of bundle. 
So what do you prefer `BUNDLE_CSV` or just `CSV`?

Reply:
The variable is defined few lines above, I think `CVS` is clear enough